dicogis/cli/cmd_list.py

```diff
@@ -160,17 +161,19 @@ def inventory(
         dico_texts=localized_strings, language_code=language
     )
 
-    # look for geographic data files
-    if input_folder is not None:
-        geodata_find = find_geodata_files(start_folder=input_folder)
-        rich.print(geodata_find)
-
+    # output format
+    if output_format == "excel":
```

Comment:
And if not ? 

**xl_workbook** will fail later (?)

Reply:
You're right but for now it's not possible to pass other output_format. It's enforced by the enum type checking:

https://github.com/Guts/DicoGIS/pull/216/files/fa6ab4a7fe8385dafb4bb03e1a912c8306d19ceb#diff-2ee58342b86852a70a7a47c462a88435bc4f18a57d3878b904bd323b74c92a97L88-L96

Reply:
Just added an additional check.